### PR TITLE
Change plots tables index

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-796

#### Link to staging deployment URL 

N.A.

#### Links to any Pull Requests related to this

N.A.

#### Anything else the reviewers should know about the changes here

Changing the keys for `plots-tables`

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [X] Tested locally with Inframock
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Review by ivababukova — APPROVED** [review]

File: cf/dynamo.yaml [hunk 1]
```diff
@@ -33,10 +33,10 @@ Resources:
         - AttributeName: "experimentId"
           AttributeType: "S"
       KeySchema:
-        - AttributeName: "plotUuid"
-          KeyType: "RANGE"
         - AttributeName: "experimentId"
           KeyType: "HASH"
+        - AttributeName: "plotUuid"
+          KeyType: "RANGE"
```
Comment (ivababukova) [inline cf/dynamo.yaml:39]: Hey Agi, does that mean that the order in which we specify the attributes here matters?

Comment (aerlaut) [reply cf/dynamo.yaml:39]: yes, it turns out so, as Seb's error states : 

```
Invalid key order.  Hash Key must be specified first in key schema, Range Key must be specifed second
```

Comment (aerlaut) [reply cf/dynamo.yaml:39]: I thought the order doesn't matter as it was a dictionary / map. Turns out it does,

Comment (ivababukova) [reply cf/dynamo.yaml:39]: ah right, it does seem to matter.

**Pull request merged**